openy_svg_formatter/src/Plugin/Field/FieldFormatter/OpenYSvgFormatter.php

```diff
@@ -43,7 +43,12 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
     foreach ($items as $delta => $item) {
       if ($item->entity) {
         $uri = $item->entity->getFileUri();
-        $svg = file_get_contents($uri);
+        $svg = '';
+        // Make sure the file actually exists.
+        if (file_exists($uri)) {
+          \Drupal::logger('locale')->warning('Could not open SVG file: %s', ['%s' => $uri]);
```

Comment:
How often are we going to have this code executed?
I'd like to get rid of logging file doesn't exist for performance reason

Reply:
@podarok this only runs on cache-rebuild, so it shouldn't be often.